src/main/scala/com/github/mlangc/slf4zio/api/package.scala

```diff
@@ -1,130 +1,116 @@
 package com.github.mlangc.slf4zio
 
-import scala.reflect.ClassTag
-import scala.util.Failure
-import scala.util.Success
-import scala.util.Try
-
+import org.slf4j.{Logger, LoggerFactory, Marker, MarkerFactory}
 import org.slf4j.event.Level
-import org.slf4j.Logger
-import org.slf4j.LoggerFactory
-import org.slf4j.Marker
-import org.slf4j.MarkerFactory
-import zio.Cause
-import zio.Has
-import zio.UIO
-import zio.ULayer
-import zio.URIO
-import zio.ZIO
-import zio.ZLayer
-import zio.clock
-import zio.clock.Clock
-import zio.duration.Duration
+import zio.{Cause, Clock, Duration, UIO, ULayer, URIO, ZIO, ZLayer}
```

Comment:
Can you use one line per import, to stay consistent? Ideally I should have integrated scalastyle into the build - but I did not.

Reply:
Thanks a lot for your work, I've looked through the code, and your changes look very reasonable. Please tell me as soon as the code is ready for a final review.

Reply:
Fixed. 

I'll try to find time to have last look over.

Reply:
Thanks a lot for your work!